src/components/response_error.tsx

```diff
@@ -58,96 +57,26 @@ export interface ResponseErrorProps {
     errorMessageTemplates?: { [key: string]: () => string };
     fieldNames?: { [key: string]: () => string };
 
-    popupHeaderText: string;
-    popupBodyText: string;
-    popupDownloadButtonText: string;
-    popupCancelButtonText: string;
+    popupHeaderText: string; // Deprecated
+    popupBodyText: string; // Deprecated
+    popupDownloadButtonText: string; // Deprecated
+    popupCancelButtonText: string; // Deprecated
```

Comment:
We should remove them if they're not used. Otherwise your IDE will suggest things you need but actually don't. Even optional property would be an option. But removing is best.

Reply:
I'm hesitant to be forced into updating a whole bunch of related front-end repos just for this.